### PR TITLE
Initial PR for progress update

### PR DESCRIPTION
**Pull request by jschiarizzi**
bin/plunder handles commands and uses yargs to make sure flags are correct
it will pass execution off to cryptokitties.js if you choose that as your item type target. [more coming soon]

Currently migrating calls from individual contracts to a central bin/helper.js script to deal with asynchronous callback issues and make it more readable/re-using code between asset types
 
Async calls in order:
- create web3 provider via truffle hd wallet
   - get eth balance via web3 to check it is enough [other sync checks]
     - check CK api to see if there are kitties in our price range
        - load the CK auction contract [other synchronous loading like creaking seaport]
           - make web3 call to CK auction contract, wait for callback on whether it has been bought
              - list newly bought kitty on opensea via Seaport

The majority of the code will be in bin/cryptokitties.js because I'm doing CK first.

### CODE REVIEW

**Comment by alexanderatallah** [comment]
@jschiarizzi can you add `node_modules` and `.env` to your gitignore and remove them from this PR? it's hard to parse

**Comment by jschiarizzi** [comment]
beatified code, moved more functions to helper class for better logic, and added those to the git ignore.

**Comment by alexanderatallah** [comment]
@jschiarizzi can you close and reopen this PR so that all the node_modules go away?
